eras/alonzo/impl/cardano-ledger-alonzo.cabal

```diff
@@ -67,7 +67,7 @@ library
         cardano-ledger-allegra >=1.1,
         cardano-crypto-class,
         cardano-ledger-binary >=1.0.1,
-        cardano-ledger-core >=1.5 && <1.7,
+        cardano-ledger-core >=1.6 && <1.7,
         cardano-ledger-mary >=1.1,
         cardano-ledger-shelley ^>=1.5,
```

Comment:
Here we would need at least 1.5.1.0, right? Because of `shelleyEqTxRaw`. 
But I'm not sure how this `^>=` operator works, it seems to me as if it would accept 1.5.0 too

Reply:
You are right on point again. Thank you for checking this!